tests/unit_tests/config/jasmine.json

```diff
@@ -0,0 +1,3 @@
+{
 "spec_dir": "tmp",
 "spec_files": [
 <<SPEC_FILES>>
 ],
 "helpers": [
 ],
 "stopSpecOnExpectationFailure": false,
 "random": true
 }
 

```

Comment:
Do you mean CR or LF?

Reply:
I am sorry, I have set wrong EOL. Corrected in the commit [dc45a44ae1b40c8058f73d083d4fc35f107d9004](https://github.com/polcak/jsrestrictor/pull/84/commits/dc45a44ae1b40c8058f73d083d4fc35f107d9004)